setup.py

```diff
@@ -67,6 +67,7 @@ def run_tests(self):
     entry_points={
         'console_scripts': ['labctl=lab.app.__main__:main'],
         'labctl': [
+            'envs=lab.app.environments:EnvsLister',
```

Comment:
Hmm i wonder if we should re-jig the command names.

- `labctl env <envname>` instead of `show`
- `labctl envs` shows all

It'd be cool if we could list all facts as well via some kind of *show everything* command as well.